src/common/utils.h

```diff
@@ -107,6 +107,8 @@ class Utils {
         std::string name, size_t elem_size, size_t buffer_size, void* buff);
     static void print_vec(arma::cx_fvec, std::string);
     static void print_mat(arma::cx_fmat);
+    static void print_rvec(arma::frowvec, std::string);
+    static void print_rmat(arma::fmat);
```

Comment:
Could you add comments for the two functions to explain the difference between print_vec/print_mat and print_rvec/print_mat?

Another comment is that print_vec and print_rvec (same for print_mat and print_rmat) seem to be very similar. Shall we combine them to be one function?